MkDocs/apm_8.8/docs/matrix.md

```diff
@@ -80,8 +80,21 @@ Reliability engineer can also easily config the different matrix versions, like
 
 ## Config Matrix version
 
-We have by default configuration(include the category cell color/default version etc) for matrix versions, but those configuration can also be changed.
```

Comment:
Can you add some context around why they would be changed.  For example to create  a visual queue that maps to standards for safety or other regulation to help them better prioritize assets .  Also how cell matrix color is different then health or risk score ranges.